ui/js/graphs.js

```diff
@@ -720,16 +721,44 @@ perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
   $scope.loadingTestData = false;
 
   var testInputCreated = false;
-
+  var testArray = [];
+  var series = [];
   $scope.addTestData = function () {
-    var series = _.clone($scope.selectedSeries);
-    series.projectName = $scope.selectedProject.name;
-    $modalInstance.close(series);
+      var selectedSeries = $scope.rightList
+      for (var i = 0; i < selectedSeries.length; i++) {
+          var result = testArray.filter(function (obj) {
+              return obj.name == selectedSeries[i];
+          });
+          series[i] = _.clone(result[0]); 
+          series[i].projectName = $scope.selectedProject.name;
```

Comment:
So this reminds me that we might want to add tests from several different projects in this dialog. I.e. you might start by selecting a few tests from mozilla-inbound, then switch to mozilla-central and pick a few tests from there. 

The way it's currently set up, I guess we try to associate all the tests with the currently selected project at the end, which (1) won't always work (a test might exist for one project and not the other) and (2) doesn't let us handle the use case above.

This means that you'll need to somehow associate a project with *each* selected test (probably when the test is selected), and return and process that later.